buildSrc/src/main/rocker/com/uber/okbuck/template/config/BuckDefs.rocker.raw

```diff
@@ -187,7 +187,7 @@ def okbuck_android_library(
         **kwargs
     )
 
-    if isLintEnabled():
+    if isLintEnabled() and not disable_lint and manifest is not None:
```

Comment:
Jus lt pass in manifest as a paramedic to isLintEnabled. We do not need to check disable_lint as it is templatized into the function body